server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/main/java/org/apache/james/webadmin/vault/routes/PurgeService.java

```diff
@@ -0,0 +1,35 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.webadmin.vault.routes;
+
+import org.apache.james.task.Task;
+import org.apache.james.vault.DeletedMessageVault;
+
+public class PurgeService {
+    private final DeletedMessageVault vault;
+
+    PurgeService(DeletedMessageVault vault) {
+        this.vault = vault;
+    }
+
+    Task purge() {
+        return vault.deleteExpiredMessagesTask();
```

Comment:
This servie do not add value. Please inline this in the task.

Reply:
I want it have same level with other `DeletedMessageVaultRoutes`. All services will keep reference to the `vault`

Reply:
Here the layered architecture do not add value. In other cases it did.

Reply:
I added `DeletedMessageVault` directed in `Routes`. How about it?